docs/workflow/building/mono/README.md

```diff
@@ -51,15 +51,15 @@ The build has a number of options that you can learn about using build -?.
 
 ### WebAssembly
 
-See the instructions in [../libraries/webassembly-instructions.md].
+See the instructions for [Building WebAssembly](../../libraries/webassembly-instructions.md).
```

Comment:
```suggestion
See the instructions for [Building WebAssembly](../../building/libraries/webassembly-instructions.md).
```

Reply:
Oops, thought I pushed that. Thanks for catching it.